src/shared/components/case-editor/services/wizard-page-field-to-case-field.mapper.spec.ts

```diff
@@ -15,7 +15,7 @@ describe('WizardPageFieldToCaseFieldMapper', () => {
 
   let wizardPageFieldToCaseFieldMapper: WizardPageFieldToCaseFieldMapper;
   const INCOMPLETE_SHOW_CONDITION_ONE = 'AddressLine2="test"';
-  const INCOMPLETE_SHOW_CONDITION_TWO = 'AddressLine1CONTAINS"test"';
+  const INCOMPLETE_SHOW_CONDITION_TWO = 'AddressLine1 CONTAINS"test"';
```

Comment:
What is the reason for this change? I think all operators can be without spaces in the CCD configuration... so if this was to get a test working, it might suggest that there's a problem with the parsing logic. 🤔

Reply:
As we have implemented peg parser it doesn't accept CONTAINS operator without space; space should be there between fieldname and CONTAINS operator otherwise it wont parse the condition. This doesn't applicable for =/!= operator.

Reply:
The problem is that there are existing configurations out there _without_ a space... so all those configurations will be broken. If the peg parser doesn't work without a preceding space, I'd suggest it needs pre-parsing to globally replace `"CONTAINS"` with `" CONTAINS"`. 🤷

Reply:
as we discussed implemented solution to replace "CONTAINS" with " CONTAINS"